chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -509,9 +513,11 @@ public synchronized void loadScene(RenderContext context, String sceneName,
     }
 
     // Try loading the grid unconditionally for now
-    boolean emiiterGridLoaded = loadEmitterGrid(context, taskTracker);
+    boolean emitterGridLoaded = true;
```

Comment:
Why does this default to `true` even if no emitter grid is loaded after this line?

Reply:
To not reload the world if it is not needed (well, it is true the name doesn't really convey the right meaning, i'll rename this variable)